src/interactions/warframe/Builds.js

```
@@ -511,23 +513,23 @@ export default class Builds extends Interaction {
         });
         modCollector.on('collect', modSelectionHandler);
 
         const modPageCollector = new InteractionCollector(interaction.client, {
-          interactionType: InteractionTypes.MESSAGE_COMPONENT,
-          componentType: MessageComponentTypes.BUTTON,
+          interactionType: InteractionType.MessageComponent,
+          componentType: ComponentType.Button,
           message,
           guild: interaction.guild,
           channel: interaction.channel,
         });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)